src/UID2.Client/PrivacyBits.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Specialized;
+
+namespace UID2.Client
+{
+    readonly struct PrivacyBits
+    {
+        // Bit 0 is legacy and is no longer in use
+        private const int CstgMask = 2;
+        private const int CstgOptOutMask = 4;
+        
+        private readonly BitVector32 _bits;
+
+        public PrivacyBits(int bitsAsInt)
+        {
+            _bits = new BitVector32(bitsAsInt);
+        }
+
+        // `_bits[x]` applies bit mask x to _bits, this is not indexing 
+        public bool IsCstgDerived => _bits[CstgMask];
```

Comment:
I understand your use of mask here but it's rather uninitiative as I (and maybe others) am thinking in index (as on https://github.com/IABTechLab/uid2-operator/blob/master/src/main/java/com/uid2/operator/util/PrivacyBits.java#L8 for example). Could we change the codes slightly to use index-based const definitions?

Or at the very least add more comments to say each mask corresponds to index 1/2...etc

Reply:
I'll change it an index-based solution